docs/source/usage/faq.rst

```diff
@@ -15,6 +15,22 @@ Frequently Asked Questions (FAQ)
 
 .. contents:: :local:
 
+.. _define-keyboard-shortcut:
+
+Defining keyboard shortcut for opening |TexText| dialog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. In Inkscape open :bash:`Edit` -> :bash:`Preferences` ->
+   :bash:`Interface` -> :bash:`Keyboard Shortcuts`
+
+2. Find the |TexText| entry in the :bash:`extension` category
```

Comment:
On my machine menu items are named slightly differently (`Keyboard Shortcuts` -> `Keyboard` & `extension` -> `Extensions`)

(I have 1.1-dev (bee5132fad, 2020-10-20))

![image](https://user-images.githubusercontent.com/2975991/96733313-f3830d00-13c1-11eb-8367-e0c36c3ba942.png)

Reply:
On Inkscape 1.0.1 it is `Keyboard Shortcuts` as written by me. But I misspelled `Extensions`. I fixed this.